diff --git a/node_modules/@types/gtag.js/index.d.ts b/node_modules/@types/gtag.js/index.d.ts
index 554603e..1158467 100755
--- a/node_modules/@types/gtag.js/index.d.ts
+++ b/node_modules/@types/gtag.js/index.d.ts
@@ -20,6 +20,9 @@ declare namespace Gtag {
   }
 
   interface ConfigParams {
+    page_title?: string | undefined;
+    page_location?: string | undefined;
+    page_path?: string | undefined;
     send_page_view?: boolean | undefined;
   }
 
